package.json

```diff
@@ -21,7 +22,7 @@
     "cover": "istanbul cover --print both ospec/bin/ospec",
     "release": "npm version -m 'v%s'",
     "preversion": "npm run test",
-    "version": "npm run build && git add mithril.js mithril.min.js",
+    "version": "npm run build && git add mithril.js mithril.min.js stream.js stream.min.js",
```

Comment:
Nit: use `mithril-stream.js` and `mithril-stream.min.js` instead, so it doesn't override `mithril/stream`.